composer.json

```diff
@@ -22,7 +22,7 @@
         "twig/twig": "Access the state machine in your twig templates (^2.10|^3.0)"
     },
     "require-dev": {
-        "phpspec/phpspec": "^5.0|^6.0",
+        "phpspec/phpspec": "^5.0|^6.3",
```

Comment:
Why so?

Reply:
[v6.3.0](https://github.com/phpspec/phpspec/releases/tag/6.3.0) allows PHP 8, while keeping backwards compatibility for PHP >= 7.2.